budget_formula/Code.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Calculates the total annual amount of an expense
+ * by multplying its amount by it's frequency.
+ *
+ * @param {number} amount - dollar cost of an expense.
+ * @param {number} frequency - annual, monthly, or weekly.
+ *
+ * @returns {number} - annual cost
+ */
+function ANNUAL_COST(amount, frequency) {
+    return amount * timesPerYear(frequency);
```

Comment:
This is indented one level further than what it should from here to the end of the file.

Reply:
Thank you! Changed